components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -416,6 +424,20 @@ - (void)setImageRenderingMode:(UIImageRenderingMode)imageRenderingMode {
   [self.tableView reloadData];
 }
 
+- (void)setAlignTitles:(BOOL)alignTitles {
+  _alignTitles = alignTitles;
+  BOOL addPaddingToCells = NO;
+  // Check to make sure atleast one action has an image. If not then all actions will align already
+  // and we don't need to add padding.
+  for (MDCActionSheetAction *action in self.actions) {
+    if (action.image) {
+      addPaddingToCells = YES;
+    }
+  }
```

Comment:
Please extract this for loop to a separate private method and invoke it. E.g.:

```objc
- (BOOL)anyActionHasAnImage {
  for (MDCActionSheetAction *action in self.actions) {
    if (action.image) {
      return YES;
    }
  }
  return NO;
}
self.addLeadingPaddingToCell = [self anyActionHasAnImage];
```

Reply:
Done